src/main/java/org/opentripplanner/routing/graph/Graph.java

```diff
@@ -240,6 +241,8 @@
      */
     public Map<FeedScopedId, TripPattern> tripPatternForId = Maps.newHashMap();
 
+    public Collection<TripOnServiceDate> tripOnServiceDates = new ArrayList();
```

Comment:
Should this be `Map` keyed by the id, that way we wouldn't need a separate runtime index?